elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -313,11 +321,29 @@ public FilterExpression visit(ComparisonNode node, Class entityType) {
                 } else {
                     return new NotFilterExpression(new Predicate(path, Operator.IN, values));
                 }
+            } else if (op == isnullOp) {
+                Operator elideOP = rsqlIsnull2Operator(values);
+                return new Predicate(path, elideOP);
             } else if (OPERATOR_MAP.containsKey(op)) {
                 return new Predicate(path, OPERATOR_MAP.get(op), values);
             }
 
             throw new RSQLParseException(String.format("Invalid Operator %s", op.getSymbol()));
         }
+
+        private Operator rsqlIsnull2Operator(List<Object> values) {
```

Comment:
I would have a function which just returns the correct FilterExpression predicate.  This should be called near the beginning of the visit function - before we attempt to coerce values.